guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -76,14 +77,76 @@ ifdef::katello[]
 # foreman-rake katello:upgrade_check
 ----
 endif::[]
+ifdef::foreman-deb[]
+. Update repositories in `/etc/apt/sources.list.d/foreman.list` on your {ProjectServer} to consume content for {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+deb http://deb.theforeman.org/ _My_Distribution_Codename_ {ProjectVersion}
+deb http://deb.theforeman.org/ plugins {ProjectVersion}
+----
++
+// List of supported OS for foreman-deb has to match "guides/common/modules/proc_configuring-repositories-foreman-deb.adoc"
+Replace _My_Distribution_Codename_ with `bullseye` if you run {ProjectServer} on Debian 11, `jammy` for Ubuntu 22.04, and `focal` for Ubuntu 20.04.
```

Comment:
I'd like to see all the items in the list use the same structure (parallelism). It tripped me up the first time I read this. How about this:

```suggestion
Replace _My_Distribution_Code_Name_ with the proper distribution code name based on where your run {ProjectServer}:

* `bullseye` for Debian 11
* `jammy` for Ubuntu 22.04
* `focal` for Ubuntu 20.04
```

Not sure if you really need the list.

Also, "code name" based on [Merriam Webster](https://www.merriam-webster.com/dictionary/code%20name).

Reply:
Applied.